### PR TITLE
Update Sysbox-EE documentation

### PR DESCRIPTION
**Pull request by rodnymolina**
Signed-off-by: Rodny Molina <rmolina@nestybox.com>

### CODE REVIEW

**Review by ctalledo — COMMENTED** [review]
The sysbox-ee specific changes look good, but I think the formatting changes are due to using a different markdown formatter. We normally use remark tool for the markdown formatting / linting, and we have .remarkrc files in the doc directories to control the formatting.

**Comment by rodnymolina** [comment]
I ran the `remark` linter this time and fixed a bunch of broken-link issues my previous linter didn't see. Looking good now. Will do the same in sysbox-ce later on as there are broken links there too.

```
    $ remark *
    README.md: no issues found
    distro-compat.md: no issues found
    issue-guidelines.md: no issues found
    quickstart/README.md: no issues found
    quickstart/dind.md: no issues found
    quickstart/images.md: no issues found
    quickstart/kind.md: no issues found
    quickstart/security.md: no issues found
    quickstart/storage.md: no issues found
    quickstart/systemd.md: no issues found
    user-guide/README.md: no issues found
    user-guide/concepts.md: no issues found
    user-guide/configuration.md: no issues found
    user-guide/deploy.md: no issues found
    user-guide/design.md: no issues found
    user-guide/dind.md: no issues found
    user-guide/images.md: no issues found
    user-guide/install.md: no issues found
    user-guide/kind.md: no issues found
    user-guide/limitations.md: no issues found
    user-guide/security.md: no issues found
    user-guide/storage.md: no issues found
    user-guide/systemd.md: no issues found
    user-guide/troubleshoot.md: no issues found
    $
    ```

File: README.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
--   [Filing Issues](#filing-issues)
--   [Support](#support)
--   [We want your feedback](#we-want-your-feedback)
--   [Uninstallation](#uninstallation)
--   [About Nestybox](#about-nestybox)
--   [Contact](#contact)
--   [Thank You](#thank-you)
+- [Introduction](#introduction)
+- [Free for Individual Developers, Paid for Enterprise](#free-for-individual-developers-paid-for-enterprise)
+- [Key Features](#key-features)
```
Comment (ctalledo) [inline README.md:7]: The TOC needs updating, does not match the headlines in the text

(No "Free for individual developers", "Key Features -> Features", etc)

**Review by ctalledo — APPROVED** [review]
The TOC of the main README file needs fixing, but all else looks good.

File: README.md (hunk 1 above)
Comment (rodnymolina) [reply README.md]: Mmm, was expecting Markdown linter to catch this, I guess we can't trust it that much ...

Done.

**Pull request merged**